delta-lake/src/integration/java/org/apache/iceberg/delta/TestSnapshotDeltaLakeTable.java

```diff
@@ -122,28 +129,36 @@ public TestSnapshotDeltaLakeTable(
 
   @Before
   public void before() throws IOException {
-    File partitionedFolder = temp1.newFolder();
-    File unpartitionedFolder = temp2.newFolder();
-    File newIcebergTableFolder = temp3.newFolder();
-    File externalDataFilesTableFolder = temp4.newFolder();
-    File typeTestTableFolder = temp5.newFolder();
+    File partitionedFolder = temp.newFolder();
+    File unpartitionedFolder = temp.newFolder();
+    File newIcebergTableFolder = temp.newFolder();
+    File externalDataFilesTableFolder = temp.newFolder();
+    File typeTestTableFolder = temp.newFolder();
+    File vacuumTestTableFolder = temp.newFolder();
+    File logCleanTestTableFolder = temp.newFolder();
     partitionedLocation = partitionedFolder.toURI().toString();
     unpartitionedLocation = unpartitionedFolder.toURI().toString();
     newIcebergTableLocation = newIcebergTableFolder.toURI().toString();
     externalDataFilesTableLocation = externalDataFilesTableFolder.toURI().toString();
     typeTestTableLocation = typeTestTableFolder.toURI().toString();
+    vacuumTestTableLocation = vacuumTestTableFolder.toURI().toString();
+    logCleanTestTableLocation = logCleanTestTableFolder.toURI().toString();
 
     spark.sql(String.format("CREATE DATABASE IF NOT EXISTS %s", NAMESPACE));
 
     partitionedIdentifier = destName(defaultSparkCatalog, partitionedTableName);
     unpartitionedIdentifier = destName(defaultSparkCatalog, unpartitionedTableName);
     externalDataFilesIdentifier = destName(defaultSparkCatalog, externalDataFilesTableName);
     typeTestIdentifier = destName(defaultSparkCatalog, typeTestTableName);
+    vacuumTestIdentifier = destName(defaultSparkCatalog, vacuumTestTableName);
+    logCleanTestIdentifier = destName(defaultSparkCatalog, logCleanTestTableName);
 
     spark.sql(String.format("DROP TABLE IF EXISTS %s", partitionedIdentifier));
     spark.sql(String.format("DROP TABLE IF EXISTS %s", unpartitionedIdentifier));
     spark.sql(String.format("DROP TABLE IF EXISTS %s", externalDataFilesIdentifier));
     spark.sql(String.format("DROP TABLE IF EXISTS %s", typeTestIdentifier));
+    spark.sql(String.format("DROP TABLE IF EXISTS %s", vacuumTestIdentifier));
```

Comment:
Instead of creating one subfolder for each table, I think we can just create tabes within the test, and do a `DROP DATABASE CASCADE` call in the end to remove all the tables after all tests

Reply:
Thank you for your suggestions. I refactored integration tests to create tables within the test and remove redundant sql commands.